server/controllers/finance/fiscal.js

```diff
@@ -643,3 +644,28 @@ function getPeriodCurrent(date) {
 
   return db.exec(sql, [date, date]);
 }
+
+/**
+ * @method getPeriodFiscalYear
```

Comment:
@lomamech 
the method list at [fiscalPeriod.js](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/server/controllers/finance/fiscalPeriod.js#L66) can return or not the period 0 and 13. Just pass ```{excludeExtremityPeriod : true I false}``` as parameter. if true the period 0 and 13 will not be returned.